python/pyspark/ml/param/_shared_params_code_gen.py

```diff
@@ -146,7 +146,9 @@ def get$Name(self):
         ("weightCol", "weight column name. If this is not set or empty, we treat " +
          "all instance weights as 1.0.", None, "TypeConverters.toString"),
         ("solver", "the solver algorithm for optimization. If this is not set or empty, " +
-         "default value is 'auto'.", "'auto'", "TypeConverters.toString")]
+         "default value is 'auto'.", "'auto'", "TypeConverters.toString"),
+        ("varianceCol", "column name for the biased sample variance" +
```

Comment:
I think you could merge the description to fit on the first line and do the linebreak after None - would make it a bit easier to read (but double check with the pylint script of course)